src/main/resources/db/reference/modernised_darts_schema.sql

```diff
@@ -144,6 +150,7 @@
 -- retention_policy            RTP
 -- transcription               TRA
 -- transcription_comment       TRC

```

Comment:
OK, I was left confused, whether this was being introduced because perhaps "Reject" comments were not being found in the same place (in heritage darts) as the initial comment at the time of making the transcription request.